### PR TITLE
Docs: User and dev doc for basic merger

### PR DESCRIPTION
**Pull request by Haruyzal**
Please have a look to ensure whether we have the same understanding of the merger :)

### CODE REVIEW

File: docs/user_docs/merger_usage.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+The **Decentralized Tech Radar** has a Merger feature to combine the contents of multiple CSV spec-files.
+
+**Note:** The Merger does not currently handle duplicated content.
+
+## How to use
+
+When wanting to merge `file0.csv` and `file1.csv`, run the following command:
+
+```cmd
+go run main.go merge <path-to-file> <path-to-other-file>
```
Comment (August-Brandt) [inline docs/user_docs/merger_usage.md:16]: The command to run the merger needs to be:
`go run main.go -merge "<path-to-file> <path-to-other-file>"`

**Review by August-Brandt — CHANGES_REQUESTED** [review]
Since `readCsvContent` is an implementation detail and not publicly facing I don't think it should be included in the documentation.

Don't think the merger functionality will be callable for the user and will just be called from other parts of the program. So the user documentation should not be needed. It's fine to have for now, since the functionality is accessible for the user. Just keep in mind that it might need to go bye bye at some point

**Comment by August-Brandt** [comment]
Except for the requested changes it all looks good

File: docs/user_docs/merger_usage.md (hunk 1 above)
Comment (Haruyzal) [reply docs/user_docs/merger_usage.md:16]: Oh interesting, I could run it in Powershell without neither the hyphen nor encasing the path-files within quotation-marks. Might there be several ways of doing running the merger?

**Comment by Haruyzal** [comment]
> Since `readCsvContent` is an implementation detail and not publicly facing I don't think it should be included in the documentation.

Oh, I thought I'd add this to the dev docs so future devs could read about the `readCsvContent` implementation. Should we delete it regardless?

File: docs/user_docs/merger_usage.md (hunk 1 above)
Comment (August-Brandt) [reply docs/user_docs/merger_usage.md:16]: That might be fine then. Will give it a try when I get home after work

File: docs/dev_docs/merger.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+This feature was developed to aggregate multiple specification files[[1]](#1-user-docs--formatting-specification-files) into one file.
+
+This can be used to create an overview of e.g. an entire department's tech stack, which can be used for reviews, analysis, or other types of breakdowns.
+
+#### [1] [User Docs: Formatting specification files.](../user_docs/spec_file_format.md)
+
+## Functions
+
+The merger currently has two functions:
```
Comment (JonasSkjodt) [inline docs/dev_docs/merger.md:15]: the merger has 3 functions, but i guess getHeader is less relevant than the two others (?). Regardless, it should probably be changed to "The merger has several functions, amongst them are.." or something

**Review by JonasSkjodt — CHANGES_REQUESTED** [review]

File: docs/dev_docs/merger.md (hunk 2 above)
Comment (Alex-itu) [reply docs/dev_docs/merger.md:15]: kinda agree, but then again I'm not completely sure what it does

Comment (Haruyzal) [reply docs/dev_docs/merger.md:15]: I'm a bit confused as I seem to have overlooked this getHeader function - in the merger.go file, I can find the passed argument header and its usages, and in the main_test.go, I see the header variable.

Can you maybe pinpoint me to this getHeader function? 🥺

Comment (JonasSkjodt) [reply docs/dev_docs/merger.md:15]: There was a small continued conversation on discord on this, did it resolve the matter @Haruyzal ?

Comment (Haruyzal) [reply docs/dev_docs/merger.md]: @JonasSkjodt Yes! Please have a look at the update :)

File: docs/user_docs/merger_usage.md (hunk 1 above)
Comment (Haruyzal) [reply docs/user_docs/merger_usage.md:20]: @August-Brandt I've added your command to the doc as well.

When you've got time: Did my command work on your machine? :)

**Review by JonasSkjodt — APPROVED** [review]
Looks good to me, well done.

File: docs/user_docs/merger_usage.md (hunk 1 above)
Comment (August-Brandt) [reply docs/user_docs/merger_usage.md:20]: I have tried the `go run main.go merge <file path> <file path>` command now. It doesn't work in my terminal. At least not on my win 10 machine. It only works with `go run main.go -merge "<file path> <file path>"`

Comment (Haruyzal) [reply docs/user_docs/merger_usage.md:20]: I'll get Theis to test `go run main.go merge <path-to-file> <path-to-other-file>` on his Linux machine, and update the docs according to our findings! :)

*Update pending.*

**Pull request merged**